projects/optic/src/__tests__/integration/capture.test.ts

```diff
@@ -113,6 +113,25 @@ describe('capture with requests', () => {
       ).toMatchSnapshot();
     });
 
+    test('handle servers not at root of host', async () => {
+      process.env.SERVER_PREFIX = '/api';
```

Comment:
just to make sure im not missing something. is the env var just for test convenience, or is SERVER_PREFIX a valid way to set the prefix when running optic?

Reply:
This is for test convenience - so that the server that I've build will only respond with prefixes

Reply:
ok, cool, that's what it looked like 👍